root/static/scripts/common/utility/formatDatePeriod.js

```diff
@@ -12,32 +12,32 @@ import ko from 'knockout';
 import formatDate from './formatDate';
 
 function formatDatePeriod<+T: {...DatePeriodRoleT, ...}>(entity: T) {
-  let {begin_date, end_date, ended} = entity;
+  let {beginDate, endDate, ended} = entity;
 
-  begin_date = formatDate(begin_date);
-  end_date = formatDate(end_date);
+  beginDate = formatDate(beginDate);
+  endDate = formatDate(endDate);
   ended = (ko.unwrap(ended): boolean);
 
-  if (!begin_date && !end_date) {
+  if (!beginDate && !endDate) {
     return ended ? l(' \u2013 ????') : '';
   }
 
-  if (begin_date === end_date) {
-    return begin_date;
+  if (beginDate === endDate) {
+    return beginDate;
   }
 
-  if (begin_date && end_date) {
-    return texp.l('{begin_date} \u2013 {end_date}', {begin_date, end_date});
```

Comment:
Revert the change to the translated string here (we use snake_case for those, and changing it means it'd have to be re-translated by our users). The second argument (the string substitutions) should look like `{begin_date: beginDate, end_date: endDate}`.